### ISSUE TITLE
Text detection using TensorFlow

### ISSUE THREAD

**Issue by uomar8** [firemaples/EverTranslator#248]
In the past I suggested to use Opencv to detect text 

but I founded a better way to do it using TensorFlow pre-trained EAST model

https://github.com/sayakpaul/Adventures-in-TensorFlow-Lite/blob/master/EAST_TFLite.ipynb

What do you think of this idea?

**Comment by firemaples** [comment]
I think it is great to be implemented to an app, but the Google ML kit used in this application is also using a pre-trained TensorFlow model.
I'll try to investigate how to use Tensorflow lite in an Android application.

**Comment by uomar8** [comment]
> 
So the solution was already in the app 🥲 

Is it possible to use Google ML kit to draw the select green rectangular automatically

**Comment by firemaples** [comment]
> > 
> 
> So the solution was already in the app 🥲
> 
> Is it possible to use Google ML kit to draw the select green rectangular automatically

Yes, but I'm not sure how to present the UI if there're several text regions on the screen.

**Comment by uomar8** [comment]
> > > 
> > 
> > 
> > So the solution was already in the app 🥲
> > Is it possible to use Google ML kit to draw the select green rectangular automatically
> 
> Yes, it is possible, but I'm not sure how to present the UI if there're several text regions on the screen.

Here's is what I think

![IMG_20230124_155926](https://user-images.githubusercontent.com/7046347/214298204-6f853866-ea43-4119-b0c1-bb95fcf5396f.jpg)

1- hide language button because here it won't be used

2- new button to select the text by Text recognition by ML kit ( it will select by block) 

3- Once user click inside it will automatically draw the green box around the selected block :) and it could be edited same as it was drawn by swap two fingers
